src/plugins/getConfig.ts

```diff
@@ -11,12 +11,6 @@ export async function getConfig (): Promise<EnvConfigI> {
     return Promise.reject(new Error('Missing environment variables'))
   }
 
-  // get and store account id, if present
-  const accountId = new URLSearchParams(windowLocationSearch).get('accountid')
-  if (accountId) {
-    sessionStorage.setItem('ACCOUNT_ID', accountId)
-  }
-
```

Comment:
Also delete unused variables.

Reply:
Right. I thought I removed `windowLocationSearch`, but I didn't. It's now removed. Thanks.